root/edit/details/AddRelationshipType.js

```diff
@@ -0,0 +1,169 @@
+    +entity1_type: CoreEntityTypeT,
+    +link_phrase: string,
+    +long_link_phrase: string,
+    +name: string,
+    +orderable_direction?: number,
+    +reverse_link_phrase: string,
+  },
+};
+
+type Props = {
+  +edit: AddRelationshipTypeEditT,
+};
+
+const AddRelationshipType = ({edit}: Props): React.Element<'table'> => {
+  const display = edit.display_data;
+  const entity0Type = ENTITY_NAMES[display.entity0_type]();
+  const entity1Type = ENTITY_NAMES[display.entity1_type]();
+  const rawIconSection = (
+    <>
+      {' '}
+      <IntentionallyRawIcon />
+    </>
+  );
+
+  return (
+    <table className="details add-relationship-type">
+      <tr>
+        <th>{addColonText(l('Name'))}</th>
+        <td>
+          {display.name}
+          {rawIconSection}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{addColonText(l('Description'))}</th>
+        <td>
+          {nonEmpty(display.description)
+            ? (
+              <>
+                {display.description}
+                {rawIconSection}
+              </>
+            ) : lp('(none)', 'description')}
+          {}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('entity0 type:')}</th>
+        <td>{entity0Type}</td>
+      </tr>
+
+      <tr>
+        <th>{l('entity1 type:')}</th>
+        <td>{entity1Type}</td>
+      </tr>
+
+      <tr>
+        <th>{l('Link phrase:')}</th>
+        <td>
+          {display.link_phrase}
+          {rawIconSection}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('Reverse link phrase:')}</th>
+        <td>
+          {display.reverse_link_phrase}
+          {rawIconSection}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('Long link phrase:')}</th>
+        <td>
+          {display.long_link_phrase}
+          {rawIconSection}
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('entity0 cardinality:')}</th>
+        <td>
+          <Cardinality cardinality={display.entity0_cardinality || 0} />
+        </td>
+      </tr>
+      <tr>
+        <th>{l('entity1 cardinality:')}</th>
+        <td>
+          <Cardinality cardinality={display.entity1_cardinality || 0} />
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('Orderable direction:')}</th>
+        <td>
+          <OrderableDirection direction={display.orderable_direction || 0} />
+        </td>
+      </tr>
+
+      {display.attributes.length > 0 ? (
+        <tr>
+          <th>{addColonText(l('Attributes'))}</th>
+          <td>
+            <ul>
+              {display.attributes.map((attribute, index) => (
+                <li key={'attribute-' + index}>
+                  {addColon(l_relationships(attribute.typeName))}
+                  {' '}
+                  {attribute.min}
+                  {'-'}
+                  {attribute.max}
+                </li>
+              ))}
+            </ul>
+          </td>
+        </tr>
+      ) : null}
+
+      <tr>
+        <th>{l('Documentation:')}</th>
```

Comment:
`:` can be saved here and both versions are localizable already:

```suggestion
        <th>{addColonText(l('Documentation'))}</th>
```